source/userguide/dm.rst

```diff
@@ -659,6 +658,106 @@ For the use case of transferring money in a banking assistant application, the f
                    "{account_from} to a {account_to}. Is that right?"]
         responder.reply(replies)
 
+
+Subset forms and invoking slot-filling within in a dialogue handler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To support dynamically iterating through a form, we suggest the use of subset forms. Consider a case in which the initial form fetches a certain set of entities, and the flow ahead depends on the values of those captured entities. This requires capturing of further entities based on those values. For this, we recommend using subset forms.
```

Comment:
why call these "subset forms"? it sounds to me like it would mean a form that only has a subset of the entities in another form. "subform" would work better.

Reply:
That's a great suggestion. I updated this and addressed your other comments too! @serapio